content/tbd/STRP.md

```diff
@@ -0,0 +1,38 @@
+---
+created: 2023-12-07
+contributors: Matthew McCarthy
+title: STRP
+url: https://strp.nl/
+locations:
+  - Eindhoven, NL
+start: "2006"
+end: 
+size: " 4-10"
+image: 
+sectors: Arts and Culture
```

Comment:
i'm assuming you put this in as this one was in the spreadsheet and so this classification had already been done. otherwise for now wouldn't focus on fillin g in sector and activity unless it is obvious and we are sure of it